index.html

```diff
@@ -11,9 +11,223 @@
+    <div class="content">
+      <div class="program-info" id="program">
+        <h6>"Where spoken words become powerful!"</h6>
+        <h1>SPOKEN WORDS AND MENTAL HEALTH | CONFERENCE 2021</h1>
+        <P class="program-definition">More often than not, we speak without actually knowing the implications of our speech, 
+          We invite you to participate in the three-day conference, send in your key notes, 
+          have the stage, tell your experiences and get feedbacks. This is also a moment of reflection to see how 
+          this has affected you and who your speech has affected and find ways to curb this harmful behaviour.</P>
+        <div class="dateandlocation" >
+          <h3> 2021.17.10(SUN) ~ 20(WED)</h3>
+          <p class="location">@ Sheraton Hotel and Suites, Abuja</p>
+        </div>
+      </div>
+    </div>
+    <!-- SECOND SECTION HOLDING THE LIST OF EVENT AND ORDER -->
+    <section class="main-program" id="project">
+      <h2 class="main-program-heading">
+        Main Program
+      </h2>
+      <!-- LIST OF ACTIVITIES IN THE EVENT -->
+      <ul class="list-group d-lg-flex flex-lg-row program-list">
+        <li class="list-group-item d-lg-flex flex-lg-column program-item">
+          <i class="fas fa-headset program-icon"></i>
+          <p class="program-item-title">Keynote Speech</p>
+          <p class="list-para">Listen to renowed Psychologists talk about personal experiences.</p>
+        </li>
+        <li class="list-group-item d-lg-flex flex-lg-column program-item">
+          <i class="fas fa-book-open program-icon"></i>
+          <p class="program-item-title">Reseach papers</p>
+          <p class="list-para">Submit your Research paper for free professional review by the present professionals.</p>
+        </li>
+        <li class="list-group-item d-lg-flex flex-lg-column program-item">
+          <i class="fas fa-walking program-icon"></i>
+          <p class="program-item-title">Walking away</p>
+          <p class="list-para">Control is part of what makes and destroys you, we can teach you to contol control</p>
+        </li>
+        <li class="list-group-item d-lg-flex flex-lg-column program-item">
+          <i class="fas fa-user-friends program-icon"></i>
+          <p class="program-item-title">Seminar</p>
+          <p class="list-para">Partner organized skills building workshops</p>
+        </li>
+        <li class="list-group-item d-lg-flex flex-lg-column program-item" id="join">
+          <i class="far fa-flag program-icon"></i>
+          <p class="program-item-title">Nigeria and Mental Health</p>
+          <p class="list-para">The race and fight will only get better when fought as one</p>
+        </li>
+      </ul>
+      <!-- REGISTRATION BUTTON -->
+      <div class="d-flex justify-center">
+        <button class="button register-button" type="button">Join Psych Conference</button>
+      </div>
+    </section>
```

Comment:
- [ ] Based on the original design, the programs in this section should all have the same width, just like they have the same height. Kindly fix this.

Reply:
Well done on this part 💯